helm_deploy/values-prod.yaml

```diff
@@ -0,0 +1,17 @@
+---
+# Per environment values which override defaults in hmpps-incentives-api/values.yaml
+
+generic-service:
+  replicaCount: 4
+
+  ingress:
+    host: incentives-api.hmpps.service.justice.gov.uk
+    contextColour: green
```

Comment:
would have been better putting `contextColour` in values.yaml and then you wouldn't need to set here